com.unity.netcode.gameobjects/Runtime/Serialization/FastBufferReader.cs

```diff
@@ -1310,5 +1323,24 @@ public unsafe void ReadValueSafe<T>(out T value, FastBufferWriter.ForFixedString
             value.Length = length;
             ReadBytesSafe(value.GetUnsafePtr(), length);
         }
+
+
+        /// <summary>
+        /// Read a FixedString value.
+        ///
```

Comment:
nit-pick?
good place for a <remarks></remarks> as opposed to the black comment line?

Reply:
For a...?

Reply:
Sorry, forgot to mark the remarks as code...